.github/workflows/osv-scanner-schedule-scan.yml

```diff
@@ -0,0 +1,44 @@
+# This file is adapted from https://github.com/google/osv-scanner
+
+
+name: OSV-Scanner Scheduled Scan
+
+on:
+  pull_request:
```

Comment:
This is weird name for a workflow, since its not only scheduled, but also triggers on pull requests and merges. We need a better name for this.